### ISSUE TITLE
USING Android 4.4.2: Got unrecognized command>FATAL:ERROR: Cannot open TUN

### ISSUE THREAD

**Issue by GoogleCodeExporter** [schwabe/ics-openvpn#226]
```
Used the same Config with success on my Asus TF101 Tab with Stock Rom and
KATKiss 4.3.

Now upgraded to KatKiss-KitKat ROM - Android 4.4.2 - Asus Transformer TF101 
public.timduru.org/Android/tf101/KatKiss/4.4/

and now I can't connect any more:
"Got unrecognized command>FATAL:ERROR: Cannot open TUN"

Evan after joining the Beta Channel and using OpenVPN für Android 0.6.4

See Log attached.

```

Original issue reported on code.google.com by `markus...@gmail.com` on 13 Jan 2014 at 8:01

Attachments:
- [log-442.txt](https://storage.googleapis.com/google-code-attachments/ics-openvpn/issue-226/comment-0/log-442.txt)

**Comment by AlmogBaku** [comment]
@schwabe is there any other solution for that? It happened to me a few times... maybe even showing a prompt that tell the user to reboot.

**Comment by AlmogBaku** [comment]
I can approve it also happened in Android 5.0.2

**Comment by schwabe** [comment]
@AlmogBaku I don't know of any other solution. As for 5.0.2 the bug that forces you to reboot should be fixed.

**Comment by AlmogBaku** [comment]
I would love to investigate this, and to help.
Is there any chance we can open a [gitter](http://gitter.im) channel for the development :) ?

**Comment by schwabe** [comment]
I have no idea what gitter is. For the bug: Feel free too look through logcat etc. to see if you can get something from that. Other than that, in most cases this error points to a broken ROM.

**Comment by AlmogBaku** [comment]
I've used it on a new formatted device, already looked in the logcat.
I'll update here on any progress.. (can you please reopen this issue please?)

About gitter- it's like a IRC(chat) channel for github project.. It will allow people to communicate easily, it's very popular over some OS projects.. take a look at http://gitter.im

**Comment by schwabe** [comment]
If you find any information that can be workaround fix in openvpn itself you should open a new issue. I don't like mixing different similar issues in one issue.

**Comment by AlmogBaku** [comment]
Sounds reasonable. I'll do so, thanks pal :)

**Comment by mixaz** [comment]
It happened to me on Genymotion emulator with Android 4.4.4 ROM (for device Google Nexus 7).  While on a 4.2.2 ROM it works OK.

Genymotion is an x86 emulator which can be downloaded for free here: www.genymotion.com

logs with full openvpn output:
4.2.2: http://pastebin.com/DeX5enYd
4.4.4: http://pastebin.com/C2wNqG9U

**Comment by schwabe** [comment]
Genymotion is broken then. Note that also the official emulator in 4.3-4.4 is broken.

**Comment by shayanhimself** [comment]
I have the same issue, which happens only sometimes. So in general it workes on all types of devices, but suddenly it disconnects from the server and cannot connect again because it cannot open Tun again. But when I wait until the openvpn process finishes, and the manually reconnect again, it opens the Tun.
